### ISSUE TITLE
Status of the project?

### ISSUE THREAD

**Issue by mauromol** [groovy/groovy-eclipse#279]
Hello,
I'm glad to see some new activity on this project, thanks to Eric Milles!

I was wondering whether the project can be considered supported again.
Also, if so, I remember Andy Clement taking a dump of the old Greclipse JIRA at CodeHaus. There, I opened many issues on this plugin, spending a moderate effort (at least from my own point of view ;-P), so I was wondering whether they could be recovered in some way and migrated here... I think many of them still apply.

Thanks in advance!

**Comment by eric-milles** [comment]
May I propose that you start from your highest priority issues, retest with the current snapshot and post here?  I have found that if you Google for "GRECLIPSE JIRA" and the number or some key terms, there are some archives out there.

I may be able to fix a few issues, but a massive deluge will definitely not be helpful for me.

**Comment by mauromol** [comment]
Yes, I see, the main problem is that I can't remember what I reported and what I did not, right now the net effect is that whenever I find a bug in Greplise I simply try to find a workaround with frustration, especially since the old JIRA archive seems to be completely lost... I hope you understand me.

Some way at least for us to browse the old archive and proactively decide whether to port some issues here or not would help... With just Google and some guess on queries it's quite hard.

**Comment by eric-milles** [comment]
Can you try Googling "GRECLIPSE JIRA [your name or username]"?  Otherwise, you mention that you are running into specific probelms.  Could you start with the worst offenders and enter them on GitHub?

**Comment by eric-milles** [comment]
I understand your concerns.  Unfortunately, I have no access to the old JIRA server.  I'm not sure a snapshot was taken before it was shut down.

**Comment by aclement** [comment]
Sorry - I just noticed this. Yes I have (somewhere) a dump of the entire contents of the old jira. If you want to see it Eric I can share it with you.

**Comment by eric-milles** [comment]
Is it possible for @mauromol<https://github.com/mauromol> to see it as well?  I think he'd like to say which issues he submitted and what priority they may have.

**Comment by mauromol** [comment]
Unfortunately searching with Google is a no way, I just find some very old traces of reports mirrored by some kind of mailing lists, but they are hardly readable and very few.

**Comment by aclement** [comment]
There is a link: https://drive.google.com/file/d/0BxM1nows33aTbV8yYU1tbWxsdnM/view

**Comment by mauromol** [comment]
This is certainly useful, thank you Andy! :)

**Comment by mauromol** [comment]
I started to review the old issues I filed. So far, I'm glad to see that many of them are fixed with current Greclipse (they were GRECLIPSE-1801, GRECLIPSE-1799, GRECLIPSE-1798 and GRECLIPSE-1797). Another couple of them are not easily reproducible and probably fixed (GRECLIPSE-1275 and GRECLIPSE-1823).
Another couple of them made me open some new issues here on GitHub. I will slowly keep on scanning the exported XML and report here anything still relevant..
